src/plugins/intel_cpu/src/nodes/executors/matmul_config.hpp

```diff
@@ -4,13 +4,29 @@
 
 #pragma once
 
+#include <cstdint>
+#include <vector>
+
 #include "executor_config.hpp"
+#include "post_ops.hpp"
 
 namespace ov::intel_cpu {
 
 struct MatMulAttrs {
-    bool transposeA;
-    bool transposeB;
+    bool transposeA = false;
+    bool transposeB = false;
+    bool withBias = false;
+    bool weightsNonTransposed = false;
+    bool sparseWeights = false;
+    uint64_t dynamicQuantizationGroupSize = 0;
```

Comment:
Nit. If we move `dynamicQuantizationGroupSize`  behind bool declarations, we can reduce the size of  `MatMulAttrs` struct by 8 bytes, due to data alignment:
```cpp
struct MatMulAttrs {
    bool transposeA = false;
    bool transposeB = false;
    bool withBias = false;
    bool weightsNonTransposed = false;
    bool sparseWeights = false;

    bool constantWeights = false;
    bool fcSemantic = false;

    uint64_t dynamicQuantizationGroupSize = 0;

    // DQ scales for quantization
    std::vector<float> dqScales;

    // Post-operations for fused operations
    PostOps postOps;
};
```

Reply:
Applied